mu/interface.py

```diff
@@ -562,6 +563,18 @@ def __init__(self, parent):
                        tool_text="Show help about Mu in a browser.")
         self.addAction(name="quit", tool_text="Quit Mu.")
 
+    def setCompactMode(self, toggle):
+        """
+        Option for a compact button bar to be used when the window is very small.
+
+        """
+        if toggle == True:
+            self.setIconSize(QSize(25, 25))
+            self.setToolButtonStyle(0)
```

Comment:
I think even at low resolutions we should keep the text (button style 3) since it is important to ensure the buttons purpose is clear.